unit/solvers/miniBDD/miniBDD.cpp

```diff
@@ -356,7 +356,7 @@ SCENARIO("miniBDD", "[core][solver][miniBDD]")
     {
       std::ostringstream oss;
       oss << format(t.as_expr());
-      REQUIRE(oss.str() == "(¬a) ∨ b");
+      REQUIRE(oss.str() == "¬a ∨ b");
```

Comment:
I believe the changes in this file are in the wrong commit as they appear to relate to precedence. But I could be wrong.

Reply:
Split into separate commit